src/theme/pages/PageWithPosts.tsx

```diff
@@ -43,11 +48,23 @@ export const PageWithPosts = ({
         as="h1"
         color="primary"
         fontSize={[6, 8]}
-        mb={[5, 5, 6]}
+        mb={[3, 4, 4]}
         mt={[5, 5, 6]}
       >
         {containerTitle}
       </Heading>
+      <Heading
+        as="p"
+        color="primary"
+        fontSize={[2, 4]}
+        mb={[3, 5]}
+        textAlign="center"
+        style={centerHorizontally}
+      >
+        <Text width={[400, 600, 700]} key="presentation">
+          {description}
+        </Text>
+      </Heading>
```

Comment:
There is a problem. The cards take the same witdh that this text.
![image](https://user-images.githubusercontent.com/4921914/143588494-5844fb0b-a93c-4fb6-a5ba-75201c3b5716.png)

![image](https://user-images.githubusercontent.com/4921914/143588453-0cb82056-07e7-47b1-b4cf-5aac861d7fd2.png)

Reply:
Also, I would prefer to simplify this code and the rendered objects in the DOM, like: 
```suggestion
      <Text
        as="p"
        width={['85%', '80%', '70%']}
        color="primary"
        fontSize={[2, 4]}
        mb={[3, 5]}
        textAlign="center"
        style={centerHorizontally}
      >
        {description}
      </Text>
```
It also is recommended to use 'rem' or '%' in place of 'px' in CSS.
It prevents some effect like:
| px | % |
|----|----|
| <img width="368" alt="image" src="https://user-images.githubusercontent.com/4921914/143590351-62c7b3b5-81b4-416f-896f-3eab732a22ec.png"> |  <img width="368" alt="image" src="https://user-images.githubusercontent.com/4921914/143590287-b789d70d-6bb4-4e3e-b640-4d911ea6d3fc.png"> |

Reply:
I am ok for using percentage instead of other configuration. I took the width values from existing we had from the site, so this should be applied everywhere.

Reply:
done in e4453ee

Reply:
In a future PR, I'll change all the unit for the different CSS styles 😉